js/main.js

```diff
@@ -0,0 +1,10 @@
+$("#more").click(function(){
+    $("section").css({"position": "fixed"});
+    $("#info").animate({"left": "10%",}, 650);
+    $("#info").animate({"left": "0%",}, 300);
+});
+
+$("#back").click(function(){
+	$("section").css({"position": "absolute"});
+    $("#info").animate({"left": "-110%"}, "slow");
```

Comment:
Set animation speed to 200ms. That keeps it consistent with other Concierge animations. @ankitr, what do you think?